File: Pass-the-Pillow.py
class Solution(object):
    def passThePillow(self, n, time):
        """
        :type n: int
        :type time: int
        :rtype: int
        """
        cycle = (n - 1) * 2
        time = time % cycle
        
        if time < n - 1:
            return time + 1
        else:
            return n - (time - (n - 1))

File: Pass-the-Pillow.java
class Solution {
    public int passThePillow(int n, int time) {
        int cycle = (n - 1) * 2;
        time = time % cycle;
        
        if (time < n - 1) {
            return time + 1;
        } else {
            return n - (time - (n - 1));
        }
    }
}
